src/WebJobs.Script.WebHost/Configuration/CorsOptionsSetup.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class CorsOptionsSetup : IConfigureOptions<CorsOptions>
```

Comment:
Unit tests missing for CorsOptionsSetup  and HostCorsOptions setup. I will open a tracking issue.